release_bot/cli.py

```diff
@@ -18,30 +18,48 @@
 from pathlib import Path
 
 from release_bot.configuration import configuration
+from release_bot.exceptions import ReleaseException
 
 
 class CLI:
+
     @staticmethod
     def parse_arguments():
         """Parse application arguments"""
+
+        def run_init(_):
+            """
+            Makes the bot to run the init script when the `release_bot init`
+            """
+            configuration.init = True
+
+        def run_bot(args):
+            """
+            Makes the bot to perform normal release tasks
+            """
+            if args.configuration:
+                args.configuration = Path(args.configuration).resolve()
+                if not args.configuration.is_file():
+                    raise ReleaseException(
+                        f"Supplied configuration file is not found: {args.configuration}")
+                    if args.debug:
```

Comment:
This is deadcode and debug logging is never set.

Reply:
Hey Sorry! I just dont know how and why I did this mistake, will correct it!

Reply:
Okay while I created the function run_bot the indentation was mistakened while copying! Really sorry for it

Reply:
I just found a bug in Atom, if you move lines up or down using ctrl it just automatically changes the indentation :(